website/src/pages/webinars/code-insights-turning-your-metrics-into-action.tsx

```diff
@@ -0,0 +1,81 @@
+    },
+    {
+        name: 'Shawn King',
+        title: 'Sourcegraph Customer Engineer',
+        img: '/staff/shawn-king.png',
+        bio: 'Shawn is a customer engineer at Sourcegraph. He has helped people use technology to get insights into their work his entire career. Coming from Microsoft and a startup called Intelligent InSites, Shawn has been able to support and provide metrics to many customers across different industries. Between the times when he’s helping customers, you can catch Shawn reading science fiction and enjoying time with his family.',
+    },
+]
+
+export const Webinar: FunctionComponent<PageProps> = props => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Sourcegraph Code Insights: Turning Metrics into Action',
+            description:
+                'Join Sourcegraph Product Manager Joel Kwartler and Customer Engineer Shawn King to learn how to use Code Insights to turn your most pressing analytics questions into KPIs you can track with mere seconds of setup.',
+        }}
+        className="navbar-white"
+    >
+        <WebinarLayout
+            title="How to track what really matters to you and your team"
+            subtitle="Sourcegraph Code Insights: Turning Metrics into Action"
+            speakers={speakers}
+            form={{
+                formId: '66361163-5e08-4be3-8ab0-6590b70df69e',
+                onFormSubmitted: () => window.open('https://my.demio.com/recording/7BYqL99g'),
+            }}
+            description={
+                <section className="col-md-6 col-12">
+                    <p>
+                        Learn how to use Code Insights to turn your most pressing analytics questions into KPIs you can
+                        track with mere seconds of setup.
+                    </p>
+                    <p>
+                        Code Insights lets you transform your code into a queryable database and create insights in
+                        seconds for anything you can search for. You can track: migration progress, terraform versions,
+                        component reuse, code smells, vulnerability indicators, language usage, CI connections,
+                        documentation, ownership signals, and many more metrics found in your code.
+                    </p>
+                    <p>
+                        But when you can quantifiably track everything in your codebase, from migrations to code smells,
+                        how do you decide where to start?
+                    </p>
+                    <p>
+                        In this interactive session, Sourcegraph Product Manager Joel Kwartler and Customer Engineer
+                        Shawn King will teach you:
+                    </p>
+                    <ul>
+                        <li className="mb-2">What Code Insights is and why it's important</li>
```

Comment:
I'm hoping we can standardize our spacings (margins, line heights, and other typography) with @traceyljohnson so we don't have to specify individual classes like this.

Reply:
I agree, spacing around our html tags for standardization would be great